src/org/zaproxy/zap/extension/websocket/WebSocketProxy.java

```diff
@@ -750,9 +825,10 @@ public String toString() {
 	 * this new message.
 	 * 
 	 * @param msg
+	 * @param initiator
 	 * @throws IOException
 	 */
-	public void sendAndNotify(WebSocketMessageDTO msg) throws IOException {
+	public void sendAndNotify(WebSocketMessageDTO msg, int initiator) throws IOException {
```

Comment:
We should keep the old method (forwarding to this one), just in case an add-on is using it.